applications/RomApplication/tests/test_compressible_potiential_rom.py

```diff
@@ -0,0 +1,60 @@
+import os
+import types
+
+try:
+    import numpy as np
```

Comment:
I see you stored the test results in .npy format. Do you need numpy only for this or does the application need numpy as well?

Reply:
I mimicked the other tests in the application. I don't know if numpy is needed for the app in general. Maybe @Rbravo555 can tell us.

Reply:
It is required.